.circleci/config.yml

```diff
@@ -488,19 +479,14 @@ jobs:
           no_output_timeout: 30m
 
   spm-revenuecat-ui-ios-15:
-    <<: *base-job
-    parameters:
-      xcode_version:
-        type: string
-        default: '14.3.0'
-      install_swiftlint:
-        type: boolean
-        default: false
+    executor:
+      name: macos-executor
+      xcode_version: '14.3.0'
 
     steps:
       - checkout
       - install-dependencies:
-          install_swiftlint: << parameters.install_swiftlint >>
+          install_swiftlint: false
```

Comment:
here I just use the param directly, I feel like this is just much clearer and less error-prone since you don't have to pass in the param